package.json

```diff
@@ -20,8 +20,8 @@
     "build": "yarn clean && node . build",
     "serve": "yarn build && cd ./public && ws",
     "develop": "yarn clean && node . develop",
-    "test": "yarn clean && nyc mocha ./packages/**/test/**/**/*.spec.js --timeout 30000",
-    "test:tdd": "yarn clean && mocha --watch ./packages/**/test/**/**/*.spec.js --timeout 15000"
+    "test": "export BROWSERSLIST_IGNORE_OLD_DATA=true && yarn clean && nyc mocha ./packages/**/test/**/**/*.spec.js --timeout 30000",
```

Comment:
had to do this to keep the browserslist message from interfering in our tests that were testing for `stderr`
https://github.com/browserslist/browserslist/issues/361

Reply:
I see that in development mode as well.  

`Browserslist: caniuse-lite is outdated. Please run next command `yarn upgrade``